package.json

```diff
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
+    "mockStart": "REACT_APP_MOCK_API=true react-scripts start",
```

Comment:
sir you can also make a .env file with the value in it but tbh this is also nice to have so let's add it